student-test/assignment1/Test_validate_username.c

```diff
@@ -18,5 +18,5 @@ void test_validate_my_username()
      * TODO: Replace the line below with your code here as described above to verify your /conf/username.txt 
      * config file and my_username() functions are setup properly
      */
-    TEST_ASSERT_TRUE_MESSAGE(false,"AESD students, please fix me!");
+    TEST_ASSERT_TRUE_MESSAGE(!strcmp(my_username(), malloc_username_from_conf_file()),"USERNAMES DO NOT MATCH!");
```

Comment:
@JLDenn what happens to the string allocated from `malloc_username_from_conf_file` here?